stage1/init/init.go

```diff
@@ -500,7 +535,11 @@ func stage1() int {
 			return 6
 		}
 
-		n, err := networking.Setup(root, p.UUID, fps, privNet, localConfig)
+		if flavor == "kvm" {
+			n, err = networking.KvmSetup(root, p.UUID, fps, privNet, localConfig)
+		} else {
+			n, err = networking.Setup(root, p.UUID, fps, privNet, localConfig)
+		}
```

Comment:
to be consistent with `Teardown()`, we should pass flavor arguments  to `Setup()` and then let networking package internals to do delegation to kvmSetup